### DISCUSSION TITLE
Resident evil hd remaster?

### DISCUSSION THREAD

**Discussion by jraynerx** [praydog/REFramework#899] [General Questions] [answered]
I was just curious if you guys had any plans to get Resident Evil hd remaster to be vr compatible i love your work and would love to see that play out in VR lol

**Comment by praydog** [comment, marked as answer]
No. Not even remotely the same engine.

**Reply by jraynerx** [reply]
Alright thank you for the response thats understandable 😊 looking forward to your future mods your doing amazing work for VR its definitely appreciated
